### DISCUSSION TITLE
Envisat ASAR processing

### DISCUSSION THREAD

**Discussion by soroushyasini** [alexisInSAR/EZ-InSAR#81] [Help]
Hi, is there any guideline for processing historical Envisat data with this package?

**Comment by alexisInSAR** [comment]
Hi @soroushyasini, 

No. But if the data are in StripMap, you can add the sensor (see [this page](EZINSAR_BIN/docs/guide_new_sensors.md). 

Kind regards, 

Alexis

**Reply by soroushyasini** [reply]
sorry for late response @alexisInSAR 
eventually I found GMTSAR for this purpose ( ENVI SLC processing ) more straightforward since it has new batch processing approachs on the new version (GMTSAR 6.4).
